source/agora/consensus/protocol/Nominator.d

```diff
@@ -302,12 +302,12 @@ extern(D):
     public void receiveEnvelope (scope ref const(SCPEnvelope) envelope) @trusted
     {
         const cur_height = this.ledger.getBlockHeight();
-        if (envelope.statement.slotIndex <= cur_height)
-        {
-            log.trace("Rejected envelope with outdated slot #{} (ledger: #{})",
-                envelope.statement.slotIndex, cur_height);
-            return;  // slot was already externalized, ignore outdated message
-        }
+        //if (envelope.statement.slotIndex <= cur_height)
+        //{
+        //    log.trace("Rejected envelope with outdated slot #{} (ledger: #{})",
+        //        envelope.statement.slotIndex, cur_height);
+        //    return;  // slot was already externalized, ignore outdated message
+        //}
```

Comment:
?

Reply:
Ah I was just testing it, I'll revert this.